website/docs/r/vapp_nat_rules.html.markdown

```diff
@@ -28,9 +29,6 @@ resource "vcd_vapp_network" "vapp-net" {
   netmask            = "255.255.255.0"
   dns1               = "192.168.2.1"
 
-  firewall_enabled = true
```

Comment:
I think we should add a vcd_vapp_firewall_rules resource (without rules) to the example with a comment saying that in order for NAT rules to be enabled, FW needs to be enabled as well.

Reply:
added